CHANGELOG.md

```diff
@@ -4,6 +4,30 @@
 
 [1]: https://www.npmjs.com/package/nodejs-logging-bunyan?activeTab=versions
 
+## v0.9.2
+
+### Bug Fixes
+- fix: logged errors are reported to error reporting ([#122](https://github.com/googleapis/nodejs-logging-bunyan/pull/122))
+- chore: fix `stream` example in README ([#134](https://github.com/googleapis/nodejs-logging-bunyan/pull/134))
+
+### Dependencies
+- chore(deps): update @google-cloud/logging to 4.x ([#152](https://github.com/googleapis/nodejs-logging-bunyan/pull/152))
```

Comment:
This change is actually notable because it fixes the reported instance id being incorrect on GCE.